### PR TITLE
Fix GetResponseEvent deprecations

### PR DESCRIPTION
**Pull request by FabienPapet**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Deprecated symfony class 
| Type?             | improvement
| Category?         | IN
| BC breaks?        |  no
| Deprecations?     | no 
| How to test?      | Technical - Ci green & automated tests

### CODE REVIEW

**Review by mflasquin — DISMISSED** [review]

File: src/PrestaShopBundle/EventListener/TokenizedUrlsListener.php [hunk 1]
```diff
@@ -30,7 +30,7 @@
 use PrestaShop\PrestaShop\Adapter\LegacyContext;
 use PrestaShop\PrestaShop\Core\Feature\TokenInUrls;
 use Symfony\Component\HttpFoundation\RedirectResponse;
-use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
```
Comment (matks) [inline src/PrestaShopBundle/EventListener/TokenizedUrlsListener.php]: PHP CS Fixer requests that KernelEvent is above RequestEvent not below

**Review by matks — CHANGES_REQUESTED** [review]

File: src/PrestaShopBundle/EventListener/TokenizedUrlsListener.php (hunk 1 above)
Comment (FabienPapet) [reply src/PrestaShopBundle/EventListener/TokenizedUrlsListener.php]: Fixed

**Review by mflasquin — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by FabienPapet** [comment]
ping @PrestaShop/qa-team can we merge this ?

**Comment by florine2623** [comment]
Waiting for the automated tests to be done and green before merging it : https://github.com/florine2623/testing_pr/actions/runs/3902697817

Thanks!

**Review by florine2623 — APPROVED** [review]
@FabienPapet ready to be merged ✅ 
Thanks!

**Comment by FabienPapet** [comment]
Thank you @florine2623

**Pull request merged**